framework/tlvf/tlvf_conf.yaml

```diff
@@ -41,6 +41,11 @@ include_yaml_path: {
   "tlvf/wfa_map/tlvSteeringRequest.yaml",
   "tlvf/wfa_map/tlvSupportedService.yaml",
   "tlvf/wfa_map/tlvTransmitPowerLimit.yaml",
+  "tlvf/wfa_map/tlvChannelScanReportingPolicy.yaml",
```

Comment:
The entire list is sorted alphabetically, please keep it sorted.

Reply:
the real question here is why did someone take the effort to sort it :)

Reply:
@RanRegev changed to be alphabetically sorted

Reply:
> take the effort